templates/repo/issue/list.tmpl

```diff
@@ -81,14 +81,15 @@
 				{{end}}
 			{{end}}
 		</div>
-		<div id="issue-filters" class="issue-list-toolbar">
-			<div class="issue-list-toolbar-left">
+		<div class="ui divider"></div>
```

Comment:
IIRC, I recently removed these dividers to save vertical space. Would prefer to not have them re-introduced, unless there is a very good reason.

Reply:
Done, removed